crates/store/src/db/sql/mod.rs

```diff
@@ -534,6 +536,52 @@ pub fn upsert_accounts(
     Ok(count)
 }
 
+fn build_insert_delta(account: &Account) -> Result<AccountDelta, DatabaseError> {
```

Comment:
nit: Would be nice to comment why this workaround is needed (and when it is used)